components/library/graphite2/Makefile

```diff
@@ -62,6 +63,7 @@ install:	$(INSTALL_32_and_64)
 test:		$(TEST_32_and_64)
 
 # Auto-generated dependencies
-REQUIRED_PACKAGES += system/library
+REQUIRED_PACKAGES += $(GCC_RUNTIME_PKG)
```

Comment:
On my build env GCC_RUNTIME_PKG is not automatically detected by gmake REQUIRED_PACKAGES.
Our convention is to add additionally needed packages under a separate label, eg. # build dependency.

Reply:
like this?

Reply:
Almost, just forgot to add a line like
# Manually added build dependencies
REQUIRED_PACKAGES += $(GCC_RUNTIME_PKG)

# Auto-generated dependencies
...

Reply:
Sigh should have used preview.

Reply:
So, it's detected on SPARC, but not on x86?

Reply:
yep.

Reply:
Can you modify comment so that it describes this.

Reply:
done